Userland/Utilities/icc.cpp

```diff
@@ -30,6 +30,15 @@ static void out_optional(char const* label, Optional<T> const& optional)
         outln("(not set)");
 }
 
+static void out_curves(Vector<Gfx::ICC::LutCurveType> const& curves)
+{
+    auto indent = MUST(String::repeated(' ', 8));
```

Comment:
This seems odd, why not make it part of the outln literal like everywhere else?

Reply:
what?

Reply:
`outln("        foo", ...)` instead of `outln("{}foo", indent, ...)`, like you already do in a bunch of places. It's fine for a variable indent width I guess, but I don't see the point when hardcoding it.

Reply:
Ah, sorry I didn't get it at first. Sure, done.

The motivation was that it's doable (for me) to eyeball `"    foo` and see that it's 4 leading spaces, but `"        foo` isn't eyeballable as 8 (again, for me). And with the explicit `8` the `/*indent=*/12` more immediately makes sense.

But I agree it looks a bit weird, so I'm also happy with putting it in the literal.